gammapy/datasets/simulate.py

```diff
@@ -574,3 +574,146 @@ def run(self, dataset, observation=None):
         geom = dataset._geom
         selection = geom.contains(events.map_coord(geom))
         return events.select_row_subset(selection)
+
+
+class SimulatedObservationMaker(MapDatasetEventSampler):
+    """
+    Sample event lists for a given observation and signal models.
+
+    Signal events are sampled from the given sky models and then
+    folded with the instrument response functions.
```

Comment:
```suggestion
    Signal events are sampled from the predicted counts distribution given by the product of the sky models and the 
    expected exposure. They are then folded with the instrument response functions.
```
I propose this modification to make the process clearer.